src/main/java/net/snowflake/client/jdbc/SnowflakeStatementV1.java

```diff
@@ -145,6 +145,25 @@ public ResultSet executeAsyncQuery(String sql) throws SQLException {
     return executeQueryInternal(sql, true, null);
   }
 
+  public ResultSet getAsyncQueryResult(String queryID) throws SQLException {
```

Comment:
The POC adds a new API function `getAsyncQueryResult()` to get the multiple statement result